src/Makefile.am

```diff
@@ -537,6 +536,7 @@ libdash_common_a_SOURCES = \
   protocol.cpp \
   saltedhasher.cpp \
   scheduler.cpp \
+  script/ismine.cpp \
```

Comment:
Hmm, looks like you didn't include makefile.am changes in 3bd6522c0a18f37ef0ce4e9918bd5530c60019fb due to them being reverted in 96ddfa5fb22022e1f8c5fbb1f197b47b095d6af0